pkg/reconciler/internal/conditions/conditions.go

```diff
@@ -37,10 +37,13 @@ const (
 	ReasonErrorGettingClient       = status.ConditionReason("ErrorGettingClient")
 	ReasonErrorGettingValues       = status.ConditionReason("ErrorGettingValues")
 	ReasonErrorGettingReleaseState = status.ConditionReason("ErrorGettingReleaseState")
+
 	ReasonInstallError             = status.ConditionReason("InstallError")
-	ReasonUpgradeError             = status.ConditionReason("UpgradeError")
-	ReasonReconcileError           = status.ConditionReason("ReconcileError")
-	ReasonUninstallError           = status.ConditionReason("UninstallError")
+	// it is UpdateError in the sdk. todo: change in SDK since Upgrade shows make more sense
+	ReasonUpgradeError   = status.ConditionReason("UpgradeError")
+	ReasonReconcileError = status.ConditionReason("ReconcileError")
+	ReasonUninstallError = status.ConditionReason("UninstallError")
+
```

Comment:
@joelanford 
We have: 

```
	// TypeInitialized is used when the controller successfully initialized 
	TypeInitialized    = "Initialized"
	// TypeDeployed is used when the controller successfully deployed 
	TypeDeployed       = "Deployed"
	// TypeReleaseFailed is used when the release of the controller failed
	TypeReleaseFailed  = "ReleaseFailed"
	// TypeIrreconcilable is used when the conditional failed status was faced to deploy an controller
	TypeIrreconcilable = "Irreconcilable"
```

However, why we need I type for success and error if we have already the boolean to let us know if success or error?